src/components/ArticleStatusFilterComponent.js

```diff
@@ -11,7 +11,7 @@ import { LeftPositionedWrapper, RightPositionedWrapper } from '../styledComponen
 const StatusFilterWrapper = styled.div`
     background-color: transparent;
     min-height: 64px;
-    height: 100%;
+    flex: 1;
```

Comment:
this just makes it so that it takes up the remaining vertical space of the container it appears in